src/live_restore/live_restore_fs.c

```diff
@@ -1263,12 +1248,10 @@ __wt_live_restore_metadata_to_fh(
      * !!!
      * While the live restore is in progress, the bit count reported by in the live restore metadata
      * can hold three states:
-     *  (0)         : This means file has not yet had a bitmap representation written to the
-     *                metadata file and therefore no application writes have gone to the
-     *                destination. In theory background thread writes may have happened but unless
-     *                the tree was dirtied the metadata update was not written out.
+     *  (0)         : This means the file has not started migration.
```

Comment:
nbits provides important state information about live restore now and I think it could be confusing to newcomers. Should we add a larger comment at the top of the live restore file for better visibility?

Reply:
I'm not sure about putting it at the top of this file, but I'm on board with extra docs.
We currently have 
```c
    /*
     * The number of bits in the bitmap. We use -1 as a special case to identify when a file has
     * finished migration and no longer needs a bitmap.
     */
```
where the nbits field is defined. Is this sufficient (pending a proper README.md for the folder)?

Reply:
Yeah sounds good